TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MecanumDrive.java

```diff
@@ -114,101 +113,14 @@ kinematics.new WheelVelConstraint(PARAMS.maxWheelVel),
 
     public final Localizer localizer;
     public Pose2d pose;
-
     private final LinkedList<Pose2d> poseHistory = new LinkedList<>();
 
     private final DownsampledWriter estimatedPoseWriter = new DownsampledWriter("ESTIMATED_POSE", 50_000_000);
     private final DownsampledWriter targetPoseWriter = new DownsampledWriter("TARGET_POSE", 50_000_000);
     private final DownsampledWriter driveCommandWriter = new DownsampledWriter("DRIVE_COMMAND", 50_000_000);
     private final DownsampledWriter mecanumCommandWriter = new DownsampledWriter("MECANUM_COMMAND", 50_000_000);
 
-    public class DriveLocalizer implements Localizer {
```

Comment:
Can you put the drive localizer back here?

Reply:
done :)